packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -154,7 +153,7 @@
 
 <script lang="ts" setup>
 import { useI18n } from '@cy/i18n'
-import Button from '@packages/frontend-shared/src/components/Button.vue'
```

Comment:
Is this button still used or can we delete this code? `'@packages/frontend-shared/src/components/Button.vue'`

Reply:
To keep the PR of a decent size, I only worked on the LaunchPad for now. 
In a 2nd PR, I will process the app and delete the unused Button.